html-css/week1/styles.css

```diff
@@ -0,0 +1,246 @@
+:root {
+    --main-red: #be3144;
+    --main-gray: #303841;
+    --main-lightgray: #0d2746;
+    --main-white: #f0f0f0;
+  }
+  * {
+    margin: 0;
+  }
+  
+  body {
+    font-family: sans-serif;
+    font-weight: 400;
+    line-height: 1.4;
+    color: var(--main-white);
+  }
+  .nav-container {
+    background-image: linear-gradient(62deg, #3a3d40 0%, #050505 100%);
+  }
+  
+  .navbar {
+    position:relative;
```

Comment:
I believe navbar position will be better with `position:fixed` or` position:sticky` in this case.
When a user scrolls down to the contact section, the user can easily click on the navbar and back to the first section again.